full-service/migrations/2023-01-20-231044_add_index_to_subaddress_spend_key/down.sql

```diff
@@ -0,0 +1 @@
+-- This file should undo anything in `up.sql`
```

Comment:
Shouldn't it drop the index?

Reply:
Ideally yes, which I can add to this branch, but we haven't been adding steps for downgrading the database. Might be worth going back and adding them all.